### ISSUE TITLE
Switch authentication from Webauth to Shibboleth

### ISSUE THREAD

**Issue by micheal-cooper** [ywc/ywc#19]
CIO says that we will have Shibboleth up and running by end of March. We will need to move all OIST authentication to Shibboleth shortly thereafter.

How long will it take to redo TIDA with Shibboleth?

**Comment by topherwhite** [comment]
Shouldn't be hard. Is shibboleth working in any capacity right now such that we could build and test on tidafe1?

**Comment by micheal-cooper** [comment]
No, it is not up yet. It will be up at end of March, according to Tim. He
is just trying to schedule the complete switchover by knowing how long the
dependent systems need to leave webauth.

**Comment by topherwhite** [comment]
As long as Shibboleth still tied to the LDAP directory, YWC/Tida will be a fast switch. All of the platform-level authentication configuration is in Apache. So, it should be a very fast switch.

**Comment by micheal-cooper** [comment]
That's what I thought. Thanks.

**Comment by micheal-cooper** [comment]
Shibb is up and running well. It is being used by some sites, and the OIST groups.oist.jp server will soon make the switch. It is already working on the staging server.

I am sending you a GMail with a cc to the guy in charge of Shibb, too.
